pyproject.toml

```
@@ -27,13 +28,22 @@ apispec = "^5.1.1"
 requests = "^2.23.0"
 pydantic = "^1.8.2"
 
+docker = { version = "^5.0.3", optional = true }
+MarkupSafe = "2.0.1"
+
+[tool.poetry.extras]
+pytest = ["docker"]
```

```
>>> import pytest_nfv_test_api
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/sander/.virtualenvs/tmp-ec786a97787c754/lib/python3.9/site-packages/pytest_nfv_test_api/__init__.py", line 25, in <module>
    import pytest
ModuleNotFoundError: No module named 'pytest'
>>> 
```
```suggestion
pytest = ["docker", "pytest"]
```
You'll also need to add it to the dependencies list of course.